backend/functions/lib/controllers/case-associated/case-associated.controller.test.ts

```diff
@@ -0,0 +1,55 @@
+import { CaseAssociatedController } from './case-associated.controller';
+import { createMockApplicationContext } from '../../testing/testing-utilities';
+import {
+  NORMAL_CASE_ID,
+  NOT_FOUND_ERROR_CASE_ID,
+  THROW_UNKNOWN_ERROR_CASE_ID,
+} from '../../testing/testing-constants';
+import { NotFoundError } from '../../common-errors/not-found-error';
+import { CaseAssociatedUseCase } from '../../use-cases/case-associated/case-associated';
+
+describe('Test case-history controller', () => {
+  let applicationContext;
+
+  beforeEach(async () => {
+    applicationContext = await createMockApplicationContext({ DATABASE_MOCK: 'true' });
+  });
+
+  test('should return associated cases when getAssociatedCases is called', async () => {
+    // TODO: Need to mock the associated cases.
+    const associatedCases = [];
```

Comment:
Looks like we never got around to this TODO?

Reply:
We have a new function in MockData that can help and sample code in the UI pa11y mock API.